webserver/outcropsketch/urls.py

```diff
@@ -14,5 +15,9 @@
     }),
     path('auth/', include('authentication.urls')),
     path('courses/', include('courses.urls')),
+    path('uploads/', include('uploads.urls')),
     path('', include('frontend.urls')),
-]
+]+ static2(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
+
+if settings.DEBUG:
+        urlpatterns += static2(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
```

Comment:
What was this added for? Also please use a more descriptive name than `static2` if we really need it

Reply:
delete :)

Reply:
test if the line above (+) is necessary